src/api/docker-files/Dockerfile

```diff
@@ -3,11 +3,19 @@
 # contained rails app generating files in the git checkout with
 # some strange user...
 
-FROM openbuildservice/frontend-base
+FROM registry.opensuse.org/obs/server/unstable/container/leap151/containers/openbuildservice/frontend-base
 ARG CONTAINER_USERID
 
+# install foreman from rpms, we only need it in docker
+RUN zypper ar http://download.opensuse.org/repositories/devel:/languages:/ruby:/extensions/openSUSE_Leap_15.1/ dlre
+RUN zypper -n --gpg-auto-import-keys ref
+RUN zypper -n in -C 'rubygem(foreman)' 'rubygem(mailcatcher)'
+
+# for lint task
+RUN npm install -g jshint
```

Comment:
Why do you move this here instead?

Reply:
what would be the alternative  @hennevogel ?

Reply:
Leave it where it was, in the frontend-base container

Reply:
All of us rebuild this container all the time.

Reply:
that would require jshint to exist as obs package - you can't npm install on obs

Reply:
@hennevogel, so all the lines 9-15 should be part of frontend-base, right?

Reply:
@DavidKang we will need the following repository `devel:languages:nodejs` to be able to install `jshint`

Reply:
That package is broken, all of the packages on OBS are...

Reply:
Guess we have to life with this here now until we find a solution...